src/chains/ethereum_1/ethChainState.ts

```diff
@@ -131,26 +131,37 @@ export class EthereumChainState {
     }
   }
 
+  getTransactionHash(signedTransaction: string) {
```

Comment:
Missing comments for this function
PLEASE ALWAYS INCLUDE COMMENTS FOR PUBLIC FUNCTIONS and usually for private functions unless they are completely self-explanatory - which this isn't